src/getVersion.c

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <os.h>
+#include <os_io_seproxyhal.h>
+#include "hathor.h"
+#include "ux.h"
+
+// handleGetVersion is the entry point for the getVersion command. It
+// unconditionally sends the app version.
+void handleGetVersion(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint16_t dataLength, volatile unsigned int *flags, volatile unsigned int *tx) {
+	G_io_apdu_buffer[0] = APPVERSION[0] - '0';
```

Comment:
Maybe we should leave a comment that `APPVERSION = "X.Y.Z"`, where `X`, `Y`, and `Z` must be a single digit between 0 and 9.

Reply:
Added this comment on the Makefile, where we define `APPVERSION`.